lib/LaTeXML/Package/graphicx.sty.ltxml

```diff
@@ -32,6 +32,7 @@ DefKeyVal('Gin', 'keepaspectratio', '', 'true');
 DefKeyVal('Gin', 'clip',            '', 'true');
 DefKeyVal('Gin', 'scale',           '');
 DefKeyVal('Gin', 'angle',           '');
+DefKeyVal('Gin', 'alt',             '');
```

Comment:
Adding an empty default via:
```perl
DefKeyVal('Gin', 'alt',             '', '');
```

would also allow the keyword-only empty use, as in:
```tex
\includegraphics[alt,width=100pt]{fig1.pdf}
```

Reply:
value-less keywords are used for booleans, or assertions, like "draft"; an "alt" here without a value would seem to mean that this image is an alternative to something else. No, makes no sense.